administrator/components/com_templates/src/Model/StyleModel.php

```diff
@@ -396,9 +396,12 @@ protected function preprocessForm(Form $form, $data, $group = 'content')
         $formFile = Path::clean($client->path . '/templates/' . $template . '/templateDetails.xml');
 
         // Load the core and/or local language file(s).
+	    // Default to using parent template language constants
+        $lang->load('tpl_' . $data->parent, $client->path)
+            || $lang->load('tpl_' . $data->parent, $client->path . '/templates/' . $data->parent);
+
+	    // Apply any, optional, overrides for child template language constants
         $lang->load('tpl_' . $template, $client->path)
```

Comment:
```suggestion
        // Load the core and/or local language file(s).
        // Default to using parent template language constants
        $lang->load('tpl_' . $data->parent, $client->path)
            || $lang->load('tpl_' . $data->parent, $client->path . '/templates/' . $data->parent);

        // Apply any, optional, overrides for child template language constants
        $lang->load('tpl_' . $template, $client->path)
```
Fix PHP code style errors.